misk-slack/src/test/kotlin/slack-api/PingSlackAction.kt

```diff
@@ -0,0 +1,50 @@
+package `slack-api`
+
+import com.squareup.moshi.Moshi
+import misk.security.authz.Unauthenticated
+import misk.web.Get
+import misk.web.Response
+import misk.web.ResponseBody
+import misk.web.actions.WebAction
+import misk.web.toResponseBody
+import okhttp3.Headers
+import java.net.HttpURLConnection
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class PingSlackAction @Inject constructor(
```

Comment:
I'd lean towards this being moved to being a nested class in the `PingSlackActionTest` file. That would improve readability of the test code that it is asserting as expected based on the action body. If it's only used within the one test file than a nested class can work well. If this is used in other test classes and files then it's fine to stay here.